student.rb

```diff
@@ -6,13 +6,11 @@ def initialize(*args, classroom)
     @classroom = classroom
   end
 
-  attr_accessor :classroom
-
   def classroom=(classroom)
     @classroom = classroom
     classroom.students.push(self) unless classroom.students.include?(self)
   end
```

Comment:
- [ ] Kindly add the `attr_reader :classroom` to the Student class so that it would be possible to check the classroom for a particular student 👍